README.md

```diff
@@ -18,6 +18,22 @@ libraryDependencies += "io.laserdisc" %% "tamer" % version
 
 See [here](example/src/main/scala/tamer/example/Main.scala) for a sample application that makes use of Tamer.
 
+## End to end testing
+
+### Database module
+
+Basic manual testing is available for the code in the example module `tamer.example.Main`.
+Thi code covers getting data from a synthetic Postgres database.
+
+Make sure you have docker installed before proceeding.
+
+From the `local` folder launch `docker-compose up` (you can enter `docker-compose down`
+if you want to start from scratch). After that you should be able to access the kafka
+gui from http://localhost:8000.
+
+Start the `runDb.sh` program which contains some example environment variables.
```

Comment:
I think we miss `load.sh` (or whatever it should be renamed to) to load some synthetic values into Postgres first

Reply:
That's covered by the docker-compose.yaml

Reply:
My bad, hadn't spot it